setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="serverless-discovery-sdk",
-    version="0.0.5",
+    version="0.1.0",
```

Comment:
Similar to the JS sdk maybe this should go to 1.0?

Reply:
@croutledgeaais sounds good to me.